src/edu/washington/escience/myria/accessmethod/JdbcAccessMethod.java

```diff
@@ -425,6 +425,22 @@ public void dropAndRenameTables(final RelationKey oldRelation, final RelationKey
   }
 
   @Override
+  public void dropTableIfExistsCascade(final RelationKey relationKey) throws DbException {
+    switch (jdbcInfo.getDbms()) {
+      case MyriaConstants.STORAGE_SYSTEM_MYSQL:
+        LOGGER.warn("MySQL does not implement CASCADE");
```

Comment:
Yeah, I don't think our group uses MySQL, but it doesn't hurt to add a warning here